packages/core/src/app/shipping/ConsignmentLineItem.tsx

```diff
@@ -104,12 +107,12 @@ const ConsignmentLineItem: FunctionComponent<ConsignmentLineItemProps> = ({ cons
                     >
                         {showItems ? (
                             <>
-                                <TranslatedString id="shipping.multishipping_item_hide_items_message" />
+                                {!isMobileViewUI && <TranslatedString id="shipping.multishipping_item_hide_items_message" />}
```

Comment:
I see you updated other files, keeping here the same, is it intentional?

Reply:
Missed that. Updated https://github.com/bigcommerce/checkout-js/pull/2106/commits/342cfd7a9cb4bc48993fb3a4329722208cc0d542